app/components/work_histories/work_history_component.rb

```diff
@@ -37,7 +37,7 @@ def load_changes
         work
           .versions
           .map do |work_version|
-            decorated_work_version = Dashboard::WorkVersionDecorator.new(work_version)
+            decorated_work_version = WorkVersionDecorator.new(work_version)
```

Comment:
@rschenk Do you want to switch to the new `ResourceDecorator.decorate` pattern here?